library/spdm_requester_lib/libspdm_req_send_receive.c

```diff
@@ -672,6 +681,18 @@ libspdm_return_t libspdm_receive_spdm_response(libspdm_context_t *spdm_context,
         }
     }
 
+    #if LIBSPDM_ENABLE_MSG_LOG
+    if ((status == LIBSPDM_STATUS_SUCCESS) && (spdm_context->msg_log.mode &
+        LIBSPDM_MSG_LOG_MODE_ENABLE) != 0) {
+        if (spdm_context->msg_log.offset + *response_size > spdm_context->msg_log.buffer_size) {
+        } else {
+            libspdm_copy_mem((uint8_t *)spdm_context->msg_log.buffer + spdm_context->msg_log.offset,
+                             spdm_context->msg_log.buffer_size, spdm_response, *response_size);
+            //spdm_context->msg_log.offset += *response_size;
```

Comment:
Why not increment here? or is this for development purposes

Reply:
It gets incremented in `libspdm_req_get_measurements.c` since, on transports that don't have 1-byte resolution, the true size of the response is only known once its been parsed.